doc/api/debugger.md

```diff
@@ -100,6 +100,12 @@ To begin watching an expression, type `watch('my_expression')`. The command
 `watchers` will print the active watchers. To remove a watcher, type
 `unwatch('my_expression')`.
 
+## Options
+
+### `--port=port`
```

Comment:
Perhaps could be done in a separate PR, but we should also add `-p` and `--help`

```
> ./node inspect

Usage: ./node inspect script.js
       ./node inspect <host>:<port>
       ./node inspect --port=<port>
       ./node inspect -p <pid>
```

Reply:
That sounds good! I will try to implement it later.

Reply:
Ah, their options are already implemented internally, but not documented. I'll add them in this PR.

Reply:
Thanks for adding! 🚀